gnuradio-runtime/python/gnuradio/ctrlport/gr-ctrlport-monitor

```diff
@@ -602,53 +592,49 @@ class UpdaterWindow(QtGui.QDialog):
         self.done(0)
 
 
-class MForm(QtGui.QWidget):
+class MForm(Qt.QWidget):
     def update(self):
         # TODO: revisit this try-except block, figure out what it's doing, and if we need to keep it. at very lease makes debugging dificult
-        if True: #try:
+        try:
             st = time.time();
             knobs = self.radioclient.getKnobs([])
             ft = time.time();
             latency = ft-st;
             self.parent.statusBar().showMessage("Current GNU Radio Control Port Query Latency: %f ms"%(latency*1000))
 
-#         except Exception, e:
-#             sys.stderr.write("ctrlport-monitor: radio.get threw exception ({0}).\n".format(e))
-#             if(type(self.parent) is MAINWindow):
-#                 # Find window of connection
-#                 remove = []
-#                 for p in self.parent.mdiArea.subWindowList():
-#                     if self.parent.conns[self.uid] == p.widget():
-#                         remove.append(p)
-#
-#                 # Find any subplot windows of connection
-#                 for p in self.parent.mdiArea.subWindowList():
-#                     for plot in self.parent.plots[self.uid]:
-#                         if plot.qwidget() == p.widget():
-#                             remove.append(p)
-#
-#                 # Clean up local references to these
-#                 self.parent.conns.remove(self.parent.conns[self.uid])
-#                 self.parent.plots.remove(self.parent.plots[self.uid])
-#
-#                 # Remove subwindows for connection and plots
-#                 for r in remove:
-#                     self.parent.mdiArea.removeSubWindow(r)
-#
-#                 # Clean up self
-#                 self.close()
-#             else:
-#                 sys.exit(1)
-#             return
+        except Exception as e:
```

Comment:
hm, uncommenting this error handling code looks better

Reply:
I'm not sure if I get you right. I think it is uncommented (and in the more recent version, it might even work).